packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -372,33 +372,29 @@ class TextSelectionOverlay {
     _toolbar?.remove();
     _toolbar = null;
 
-    _handleController.stop();
     _toolbarController.stop();
   }
 
   /// Final cleanup.
   void dispose() {
     hide();
-    _handleController.dispose();
     _toolbarController.dispose();
   }
 
   Widget _buildHandle(BuildContext context, _TextSelectionHandlePosition position) {
     if ((_selection.isCollapsed && position == _TextSelectionHandlePosition.end) ||
          selectionControls == null)
       return Container(); // hide the second handle when collapsed
-    return FadeTransition(
-      opacity: _handleOpacity,
-      child: _TextSelectionHandleOverlay(
-        onSelectionHandleChanged: (TextSelection newSelection) { _handleSelectionHandleChanged(newSelection, position); },
-        onSelectionHandleTapped: _handleSelectionHandleTapped,
-        layerLink: layerLink,
-        renderObject: renderObject,
-        selection: _selection,
-        selectionControls: selectionControls,
-        position: position,
-        dragStartBehavior: dragStartBehavior,
-      )
+    return _TextSelectionHandleOverlay(
+      onSelectionHandleChanged: (TextSelection newSelection) { _handleSelectionHandleChanged(newSelection, position); },
+      onSelectionHandleTapped: _handleSelectionHandleTapped,
+      layerLink: layerLink,
+      renderObject: renderObject,
+      selection: _selection,
+      selectionControls: selectionControls,
+      position: position,
+      dragStartBehavior: dragStartBehavior,
+      vsync: Overlay.of(context),
```

Comment:
Can you not just get this with `Overlay.of` from within the _TextSelectionHandleOverlay?

Reply:
Removed since using `SingleTickerProviderStateMixin`.